credit_calculator.py

```diff
@@ -26,9 +27,14 @@ def months_until_paid_out(credit_sum, monthly_payment, initial_payment, monthly_
 
 
 if __name__ == "__main__":
-    credit_sum = 3000000
-    monthly_payment = 12500
-
+    
+    data=[]
```

Comment:
@Ghaust It would be nice to do it with [`argparse`](https://docs.python.org/3/library/argparse.html). It will allow to check if all parameters are present and also add description of all required parameters.

Also, if you look on the code again `credit_sum` and `monthly_payment` aren't used further which neglect the whole reason to have input parameters. With the library above, it would be also possible to check parameters type because now it will simply crash with parameters like `1000,0000.25`.

Reply:
Okay thank you @ChameleonTartu ! I will dive into that and do another commit.